### DISCUSSION TITLE
Preguntas Clase 3 - 5/24/2024

### DISCUSSION THREAD

**Discussion by DanteCely** [nodo-eafit-frontend/frontend-workshop-2024-1#5] [Q&A]

**Comment by Nexxies** [comment]
Profe puede volver a explicarme como abrir desde la terminal el navegador  esa parte no la pude hacer ayer por qué se me pegó el pc y me perdí.

**Reply by DanteCely** [reply]
DAle vamos a checar el tema. NO hay problema ;)

**Comment by diegolsg** [comment]
### comñeros
**si desean instalar ubuntu en windows, también pueden implementar una maquina virtual con virtual box o MVware**

**Reply by DanteCely** [reply]
No la recomiendo porque con WSL podemos resolverlo de una forma optimizada en lugar de utilizar una Máquina Virtual como la de [VirtualBox](https://www.virtualbox.org/), 

> WSL: feature of Windows that allows you to run a Linux environment on your Windows machine, without the need for a separate virtual machine or dual booting. _[What is the Windows Subsystem for Linux?](https://learn.microsoft.com/en-us/windows/wsl/about)_

Una máquina virtual nos intalaría todo un sistema operativo completo, y por el momento, solo necesitamos un ecosistema linux en terminal para trabajar.

> Virtual machines (VMs) allow a business to run an operating system that behaves like a completely separate computer in an app window on a desktop. _[virtual machines](https://www.vmware.com/topics/glossary/content/virtual-machine.html)_

**Comment by vns31407** [comment]
profe yo tengo una duda sobre como visualizar las grabaciones de las clases.
en teams tenemos acceso a algun grupo si nos dan correo institucional?

**Reply by DanteCely** [reply]
Cuando tengamos todos los correos institucionales, udaremos un espacio de la clase para revisar ese tema 😉

**Comment by Alexakano88** [comment]
Profe para relacionar los archivos entre el css y el html lo pudes repetir porfa?

**Reply by DanteCely** [reply]
[Hojas de estilo CSS externas: Cómo vincular CSS a HTML e importar al Head](https://www.freecodecamp.org/espanol/news/hojas-de-estilo-css-externas-como-vincular-css-a-html-e-importar-a-head/)
